lib/types/local-date.js

```diff
@@ -64,12 +64,13 @@ function LocalDate(year, month, day) {
     throw new Error('You must provide a valid year, month and day');
   }
 
-  //If date cannot be represented yet given a valid days since epoch, track
-  //it internally.
-  const value = isNaN(this.date.getTime()) ? year : null;
-  Object.defineProperty(this, '_value', { enumerable: false, value: value });
-
-  const self = this;
+  /** 
+   * If date cannot be represented yet given a valid days since epoch, track
+   * it internally.
+   * @ignore
```

Comment:
`@private` doclet prevents a property / class from being included, there is no need to add also `@ignore`.